automation/taskcluster/decision_task_pull_request.py

```diff
@@ -19,7 +19,7 @@
 REPO_URL = os.environ.get('GITHUB_HEAD_REPO_URL')
 BRANCH = os.environ.get('GITHUB_HEAD_BRANCH')
 COMMIT = os.environ.get('GITHUB_HEAD_SHA')
-PR_TITLE = os.environ.get('GITHUB_PULL_TITLE')
+PR_TITLE = os.environ.get('GITHUB_PULL_TITLE') or ""
```

Comment:
or `.get(..., "")` -- it's a dict-like thing.

Reply:
Thanks!